### PR TITLE
Add all argument to ListImages

### PR DESCRIPTION
**Pull request by xbglowx**
This is similar to ListContainers

### CODE REVIEW

File: dockerclient.go [hunk 1]
```diff
@@ -521,8 +521,12 @@ func (client *DockerClient) RemoveContainer(id string, force, volumes bool) erro
 	return err
 }
 
-func (client *DockerClient) ListImages() ([]*Image, error) {
-	uri := fmt.Sprintf("/%s/images/json", APIVersion)
+func (client *DockerClient) ListImages(all bool) ([]*Image, error) {
+        argAll := 0
+        if all == true {
```
Comment (ehazlett) [inline dockerclient.go]: Minor nit: this could just be
```
if all {
    argAll = 1
}
```

**Comment by xbglowx** [comment]
@ehazlett I updated the commit with your recommended change.

**Comment by ehazlett** [comment]
LGTM

/cc @vieux @donhcd

**Comment by vieux** [comment]
LGTM

**Pull request merged**